src/IssueViz.Security.UnitTests/Taint/TaintIssuesBindingMonitorTests.cs

```diff
@@ -59,23 +62,40 @@ public void Ctor_SubscribeToSolutionBindingChanged()
         {
             var synchronizer = new Mock<ITaintIssuesSynchronizer>();
             var activeSolutionBoundTracker = new Mock<IActiveSolutionBoundTracker>();
+            var folderWorkspaceInitialized = new Mock<IFolderWorkspaceInitializedEvent>();
 
-            new TaintIssuesBindingMonitor(activeSolutionBoundTracker.Object, synchronizer.Object);
+            new TaintIssuesBindingMonitor(activeSolutionBoundTracker.Object, folderWorkspaceInitialized.Object, synchronizer.Object);
 
             activeSolutionBoundTracker.Raise(x => x.SolutionBindingChanged += null, new ActiveSolutionBindingEventArgs(BindingConfiguration.Standalone));
 
             synchronizer.Verify(x => x.SynchronizeWithServer(), Times.Once);
         }
 
+        [TestMethod]
+        public void Ctor_SubscribeToFolderInitialized()
+        {
+            var synchronizer = new Mock<ITaintIssuesSynchronizer>();
+            var activeSolutionBoundTracker = new Mock<IActiveSolutionBoundTracker>();
+            var folderWorkspaceInitialized = new Mock<IFolderWorkspaceInitializedEvent>();
+
+            new TaintIssuesBindingMonitor(activeSolutionBoundTracker.Object, folderWorkspaceInitialized.Object, synchronizer.Object);
+
+            folderWorkspaceInitialized.Raise(x => x.FolderWorkspaceInitialized += null, EventArgs.Empty);
+
+            synchronizer.Verify(x => x.SynchronizeWithServer(), Times.Once);
```

Comment:
Maybe add a line to clear the synchronizer (`synchronizer.Invocations.Clear()`?) immediately before raising the event. That way the test guarantees the synchronizer is called from the event handler rather than the constructor.
Same for the other test.